### PR TITLE
Update vision queue with tuple (time, bboxes).

### LINKED ISSUE #173: Update vision queue as priority queue.

### PR DESCRIPTION
**Pull request by coledie**
Change how the vision queue is updated, simulate a priority queue to our needs.

### CODE REVIEW

File: vision/pipeline.py [hunk 1]
```diff
@@ -77,8 +81,7 @@ def run(self, prev_state):
             pass  # raise AttributeError(f"Unrecognized state: {state}")
 
         ##
-        for bbox in bboxes:
-            self.vision_communication.put(bbox, self.PUT_TIMEOUT)
+        self.vision_communication.put((self.time, bboxes), self.PUT_TIMEOUT)
```
Comment (ClayMav) [inline vision/pipeline.py]: Just send the current time in datetime format, then I can prune old ones

**Review by ClayMav — CHANGES_REQUESTED** [review]

File: vision/pipeline.py (hunk 1 above)
Comment (coledie) [reply vision/pipeline.py]: Is there any specific function I can use that would work best?

Comment (ClayMav) [reply vision/pipeline.py]: datetime.datetime() https://docs.python.org/3/library/datetime.html

Comment (coledie) [reply vision/pipeline.py]: datetime.datetime.now() it is

**Review by ClayMav — APPROVED** [review]

**Pull request merged**